src/middlewared/middlewared/plugins/account.py

```diff
@@ -1082,3 +1082,7 @@ class Config:
                     f'{schema}.users',
                     f'Following users do not exist: {", ".join(map(str, notfound))}',
                 )
+
+
+async def setup(middleware):
+    await middleware.call('user.sync_builtin')
```

Comment:
We are running this everytime middlewared starts.

Shouldnt we do something to run it only once? After install/upgrade?